composer.json

```diff
@@ -38,5 +40,11 @@
     },
     "scripts-descriptions": {
         "belt-update": "Performs update to the latest Release Belt version."
+    },
+    "extra": {
+        "assets-dir": "public/assets",
```

Comment:
I would prefer `public/css` for CSS, for my personal convention, please. :)

Reply:
Fine for me :)